src/applications/vaos/new-appointment/components/ScheduleCernerPageV2.jsx

```diff
@@ -0,0 +1,72 @@
+import ProgressButton from 'platform/forms-system/src/js/components/ProgressButton';
+import { getCernerURL } from 'platform/utilities/cerner';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import FacilityPhone from '../../components/FacilityPhone';
+import NewTabAnchor from '../../components/NewTabAnchor';
+import { TYPE_OF_CARE_IDS } from '../../utils/constants';
+import { scrollAndFocus } from '../../utils/scrollAndFocus';
+import { routeToPreviousAppointmentPage } from '../redux/actions';
+import { getChosenFacilityInfo, selectTypeOfCare } from '../redux/selectors';
+
+const pageKey = 'scheduleCerner';
```

Comment:
Hmm doesn't this need to be a different key? Otherwise it will be a duplicate of ScheduleCernerPage right?